### PR TITLE
update docs about TF importer and accelerators

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Read [release notes](https://github.com/openvinotoolkit/model_server/releases) t
 Review the [Architecture concept](https://docs.openvino.ai/2022.2/ovms_docs_architecture.html) document for more details.
 
 Key features: 
-- support for multiple frameworks, such as Caffe, TensorFlow, MXNet, PaddlePaddle and ONNX
+- support for multiple frameworks, such as Caffe, TensorFlow, MXNet, PaddlePaddle, ONNX and TensorFlow
```
Comment (mzegla) [inline README.md:18]: TensorFlow already is on that list

File: docs/accelerators.md [hunk 2]
```diff
@@ -121,15 +121,23 @@ docker run --rm -it  --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/rende
...
-> The public docker image includes the OpenCL drivers for GPU in version 21.38.21026 (RedHat) and 21.48.21782 (Ubuntu).
-
-Support for [Intel Arc](https://www.intel.com/content/www/us/en/architecture-and-technology/visual-technology/arc-discrete-graphics.html), which is in preview now, requires newer driver version `22.10.22597`. You can build OpenVINO Model server with ubuntu base image and that driver using the command below:
+GPU device can be used also on Windows hosts with Windows Subsystem for Linux 2 (WSL2). In such scenario, there are needed extra docker parameters like below:
 ```bash
-git clone https://github.com/openvinotoolkit/model_server.git
-cd model_server
-make docker_build INSTALL_DRIVER_VERSION=22.10.22597
+
+docker run --rm -it  --device=/dev/dxg --volume /usr/lib/wsl:/usr/lib/wsl --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -u $(id -u):$(id -g) \
```
Comment (mzegla) [inline docs/accelerators.md:127]: Can we break down changes between Unix and WSL version of `docker run` command? Tell users that `--device` changes and that mounting `/usr/lib/wsl` is needed and for what.

File: docs/accelerators.md [hunk 3]
```diff
@@ -121,15 +121,23 @@ docker run --rm -it  --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/rende
 
 ```
 
-> **NOTE**:
-> The public docker image includes the OpenCL drivers for GPU in version 21.38.21026 (RedHat) and 21.48.21782 (Ubuntu).
-
-Support for [Intel Arc](https://www.intel.com/content/www/us/en/architecture-and-technology/visual-technology/arc-discrete-graphics.html), which is in preview now, requires newer driver version `22.10.22597`. You can build OpenVINO Model server with ubuntu base image and that driver using the command below:
+GPU device can be used also on Windows hosts with Windows Subsystem for Linux 2 (WSL2). In such scenario, there are needed extra docker parameters like below:
```
Comment (mzegla) [inline docs/accelerators.md:124]:
```suggestion
GPU device can be used also on Windows hosts with Windows Subsystem for Linux 2 (WSL2). In such scenario, there are needed extra docker parameters. See the command below:
```

File: docs/accelerators.md [hunk 4]
```diff
@@ -240,15 +248,21 @@ THROUGHPUT
 
 ## Using NVIDIA Plugin
 
-*Note:* To build container with NVIDIA plugin use commands:
+OpenVINO Model Server can be used also with NVIDIA GPU cards. It is implemented thanks to NVIDIA plugin from the [github repo openvino_contrib](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/nvidia_plugin).
```
Comment (mzegla) [inline docs/accelerators.md:251]:
```suggestion
OpenVINO Model Server can be used also with NVIDIA GPU cards by using NVIDIA plugin from the [github repo openvino_contrib](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/nvidia_plugin).
```

File: docs/ovms_quickstart.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Quickstart Guide {#ovms_docs_quick_start_guide}
 
 OpenVINO Model Server can perform inference using pre-trained models in either [OpenVINO IR](https://docs.openvino.ai/2022.2/openvino_docs_MO_DG_IR_and_opsets.html#doxid-openvino-docs-m-o-d-g-i-r-and-opsets) 
-or [ONNX](https://onnx.ai/) format. You can get them by:
+or [ONNX](https://onnx.ai/), PaddlePaddle[https://github.com/PaddlePaddle/Paddle] or TensorFlow format [https://www.tensorflow.org/]. You can get them by:
```
Comment (mzegla) [inline docs/ovms_quickstart.md:4]:
```suggestion
, [ONNX](https://onnx.ai/), PaddlePaddle[https://github.com/PaddlePaddle/Paddle] or TensorFlow format [https://www.tensorflow.org/]. You can get them by:
```

File: docs/ovms_quickstart.md [hunk 6]
```diff
@@ -124,6 +124,7 @@ In the `results` folder, you can find files containing inference results.
 In our case, it will be a modified input image with bounding boxes indicating detected faces.
 ![Inference results](quickstart_result.jpg)
 
-Note: Similar steps can be performed with an ONNX model. Check the inference [use case example](../demos/using_onnx_model/python/README.md) with a public ResNet model in ONNX format.
+Note: Similar steps can be performed with an ONNX model. Check the inference [use case example](../demos/using_onnx_model/python/README.md) with a public ResNet model in ONNX format
+or with [TensorFlow model demo](../demos/image_classification_using_tf_model/python/README.md ).
```
Comment (mzegla) [inline docs/ovms_quickstart.md:128]:
```suggestion
or [TensorFlow model demo](../demos/image_classification_using_tf_model/python/README.md ).
```

**Review by rasapala — APPROVED** [review]
LGTM

File: docs/models_repository.md [hunk 7]
```diff
@@ -1,14 +1,18 @@
...
 - [OpenVINO IR](https://docs.openvino.ai/2022.2/openvino_docs_MO_DG_IR_and_opsets.html#doxid-openvino-docs-m-o-d-g-i-r-and-opsets), where the graph is represented in .bin and .xml files 
 - [ONNX](https://onnx.ai/), using the .onnx file
 - [PaddlePaddle](https://www.paddlepaddle.org.cn/en), using .pdiparams and .pdmodel files
+- [TensorFlow](https://www.tensorflow.org/), using frozen graph format with .pb extension (preview feature)
 
 To use models trained in other formats you need to convert them first. To do so, use 
 OpenVINO’s [Model Optimizer](https://docs.openvino.ai/2022.2/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) for IR, or different
 [converters](https://onnx.ai/supported-tools.html) for ONNX.
 
+The feature of direct import of Tensorflow models is currently a preview feature. It supports currently only the frozen graph and not all topologies can be used that way.
```
Comment (dkalinowski) [inline docs/models_repository.md:13]:
```suggestion
The feature of direct import of Tensorflow models is currently a preview feature. Currently It supports only the frozen graph and not all topologies can be used that way.
```

File: docs/ovms_quickstart.md (hunk 5 above)
Comment (dkalinowski) [inline docs/ovms_quickstart.md:4]: Missing Caffee

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**